tests/subsys/net/lib/fota_download/src/main.c

```diff
@@ -119,6 +119,7 @@ static void init(void)
 	zassert_equal(err, 0, NULL);
 }
 
+#define NO_TLS -1
```

Comment:
I suggest putting #defines in a header file.

Reply:
Only used in this file will move it to the top.